### PR TITLE
wxGUI: rename Modules tab to Tools tab, and Display tab to Layers tab

### PR DESCRIPTION
**Pull request by lindakladivova**

### CODE REVIEW

**Review by petrasovaa — CHANGES_REQUESTED** [review]
There are other places that need to be changed, specifically in SearchModuleWindow. There are several tooltips where module should be changed to tool to keep the interface consistent.

**Comment by landam** [comment]
This change should be applied before 8.0.0 release

**Comment by landam** [comment]
@lindakladivova Please change terminology also in `g.gui.modeler` (eg. menu): `Model -> Add command`.

File: gui/wxpython/gui_core/preferences.py [hunk 1]
```diff
@@ -1129,7 +1129,7 @@ def _createCmdPage(self, notebook):
         close = wx.CheckBox(
             parent=panel,
             id=wx.ID_ANY,
-            label=_("Close dialog when module is successfully finished"),
+            label=_("Close dialog when command is successfully finished"),
```
Comment (petrasovaa) [inline gui/wxpython/gui_core/preferences.py:1132]: Should be tool here as well?

Comment (lindakladivova) [reply gui/wxpython/gui_core/preferences.py:1132]: Probably yes, but I was not sure about formulation. The sentence "tool is successfully finished" seems to me a bit strange. The tool is not a process.
What about changing the sentence to : Close dialog when tool process is completed successfully?

**Comment by lindakladivova** [comment]
@petrasovaa I tried to find "Module/module/Modules/modules" occurrences that go "out" to GUI users. There are many other occurrences mainly in the class descriptions, variable names, class names.
There is also a big question about documentation, e.g. https://grass.osgeo.org/grass78/manuals/wxGUI.modules.html etc.

**Comment by lindakladivova** [comment]
One more thing - I like the sentence from the manual: GRASS GIS functionality is organized into modules, which are standalone programs with the defined interface.
This is something where the word "module" really fits since it is related to Unix theory and for CLI users who are used to this concept, the word "modules" can be clearer than "tools".
The problem we face is mainly related to newcomers who use GUI and basically do not use tools as individual modules, or maybe they use :-), but do not perceive it like that. @petrasovaa @wenzeslaus any opinions?

File: gui/wxpython/gui_core/preferences.py (hunk 1 above)
Comment (petrasovaa) [reply gui/wxpython/gui_core/preferences.py:1132]: How about "Close dialog when completed successfully"?

**Comment by wenzeslaus** [comment]
I don't think _modules_ are clearer than _tools_ here.

> ...the word "module" really fits since it is related to Unix theory and for CLI users who are used to this concept, the word "modules" can be clearer than "tools".

I don't think this applies here, at least not without a heavy GRASS experience. For example, in the "Close dialog" message, it is really a _process_ or _subprocess_ if one wants to be technical. _Modularity_ and _modular programs_ are terms mentioned in unix philosophy, but not _modules_ ([Basics of the Unix Philosophy](http://www.catb.org/~esr/writings/taoup/html/ch01s06.html), [Wikipedia: Unix_philosophy](https://en.wikipedia.org/wiki/Unix_philosophy)). On the other hand, the term _module_ is used Python, Linux Kernel, and C++20. All these refer to a component in a modular system - which applies in GRASS too - but they don't refer to something _executable_ or directly applicable to data like a _program_ or _function_. Both of these meanings are quite strong in the context of GRASS GIS; I think they are stronger, more immediate to the users, than _modularity_. _Module_ seems to be used where the _modularity_ is the main or only thing which can be said about that thing. 

> ...I like the sentence from the manual: GRASS GIS functionality is organized into modules, which are standalone programs with the defined interface.

You can do the same what the unix philosophy articles do. Talk about the _tools_ being _modular_ or, even better, _how they are modular_ instead of hoping you will convey the right thing with a word _module_.

**Comment by landam** [comment]
For this PR let's keep changes minimalist. Conceptual terminology changes ("module" -> "tool") in the whole GRASS code base including documentation would need to be discussed first. Let's change only Layer Manager tab and related GUI labels. In the documentation (`wxGUI.html`) we can explain that in the "Tools" tab GRASS modules (tools) can be searched and launched.

File: gui/wxpython/gui_core/preferences.py (hunk 1 above)
Comment (landam) [reply gui/wxpython/gui_core/preferences.py:1132]: @lindakladivova Please modify your PR accordingly.

**Comment by lindakladivova** [comment]
I would merge it and then open another PR for 

> For this PR let's keep changes minimalist. Conceptual terminology changes ("module" -> "tool") in the whole GRASS code base including documentation would need to be discussed first. Let's change only Layer Manager tab and related GUI labels. In the documentation (`wxGUI.html`) we can explain that in the "Tools" tab GRASS modules (tools) can be searched and launched.

I have added the description and a new screenshot where "Tools" are nicely captured. If you agree, this could be merged now and then we can open another PR focusing on editing https://grass.osgeo.org/grass80/manuals/wxGUI.html docs - there is a lot of outdated stuff.

**Comment by lindakladivova** [comment]
> > For this PR let's keep changes minimalist. Conceptual terminology changes ("module" -> "tool") in the whole GRASS code base including documentation would need to be discussed first. Let's change only Layer Manager tab and related GUI labels. In the documentation (`wxGUI.html`) we can explain that in the "Tools" tab GRASS modules (tools) can be searched and launched.
> 
> I have added the description and a new screenshot where "Tools" are nicely captured. If you agree, this could be merged now and then we can open another PR focusing on editing https://grass.osgeo.org/grass80/manuals/wxGUI.html docs - there is a lot of outdated stuff.

Maybe I would even let the screenshot as it was, and just change the sentence in the documentation. The screenshot does not fit into the concept unless we do not change the descriptions around.

**Review by wenzeslaus — APPROVED** [review]

**Comment by wenzeslaus** [comment]
> ...another PR focusing on editing...docs...

Yes, doing an overall update is probably the best.

**Comment by wenzeslaus** [comment]
> Maybe I would even let the screenshot as it was, and just change the sentence in the documentation. The screenshot does not fit into the concept unless we edit paragraphs around.

But perhaps an outdated screenshot is even more confusing than the outdated docs. I suggest keeping it since you already have it there.

**Review by petrasovaa — APPROVED** [review]

**Review by landam — APPROVED** [review]

**Comment by lindakladivova** [comment]
Please @petrasovaa , could you merge it? :-) I haven't the write access to the repository...

**Pull request merged**